### PR TITLE
Add explanation to "Ignored" run status

### PR DESCRIPTION
**Pull request by rmn-snpk**
Fixes OPS-1533.

## Testing Checklist

Check all that apply:

- [X] I tested the feature thoroughly, including edge cases

- [X] I verified all affected areas still work as expected

- [X] Automated tests were added/updated if necessary

- [X] Changes are backwards compatible with any existing data, otherwise a migration script is provided

## Visual Changes (if applicable)

image

### CODE REVIEW

File: packages/react-ui/src/app/features/flow-runs/lib/flow-run-utils.ts [hunk 1]
```diff
@@ -115,6 +116,11 @@ export const flowRunUtils = {
         };
     }
   },
+  getStatusExplanation(status: FlowRunStatus): string | undefined {
+    if (status === FlowRunStatus.IGNORED) {
+      return t('Previous scheduled run is still in progress');
+    }
+  },
```
Comment (rSnapkoOpenOps) [inline packages/react-ui/src/app/features/flow-runs/lib/flow-run-utils.ts:123]: We can update this function and use switch case in future
